backend/app/api/endpoints/adapter/attachments.py

```
@@ -599,3 +599,50 @@ async def get_attachment_by_subtask(
         raise HTTPException(status_code=403, detail="Access denied")
 
     return AttachmentDetailResponse.from_context(context)
+
+
+@router.get("/task/{task_id}/all", response_model=List[AttachmentDetailResponse])
+async def get_all_task_attachments(
+    task_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)